xml/deployment_installation.xml

```diff
@@ -823,14 +823,318 @@
+         install a local <command>openstack</command> client</link>.
+        </para>
+        <para>
+         To use the local client, you need to access <guimenu>Project &rarr; Compute &rarr; 
+         Access &amp; Security</guimenu> in the Horizon Dashboard and click on the 
+         <guimenu>Download &ostack; RC File v3</guimenu>.
+        </para>
+        <para>
+         The downloaded file is a script that you then need to load using the 
+         <command>source</command> command. The script will ask you for your 
+         &soc; password.
+        </para>
+         <screen>
+&prompt.user;<command>source container-openrc.sh</command>
+         </screen>
+       </note>
+       <procedure>
+
+         <step>
+          <para>
+           Upload the container image to &ostack; Glance (Image service). This
+           example uses the name <literal>CaaSP-3</literal> as the name of the 
+           image that is created in &soc;.
+          </para>
+         <screen>
+&prompt.user;<command>openstack image create --public --disk-format qcow2 \
+--container-format bare \
+--file SUSE-CaaS-Platform-3.0-OpenStack-Cloud.x86_64-1.0.0-GM.qcow2 \
+<replaceable>CaaSP-3</replaceable></command>
+         </screen>
+         </step>
+         <step>
+          <warning>
+           <title>Replace the default <literal>root_password</literal></title>
+            <para>
+             Do not use the <filename>caasp-environment.yaml</filename> directly
+             from the GitHub repository.
+            </para>
+            <para>
+             You must make sure to replace the value for 
+             <literal>root_password</literal> with a secure password.
+             This will become the password for the &rootuser; account on all 
+             nodes in the stack.
+            </para>
+         </warning>
+          <para>
+           Download the <filename>caasp-stack.yaml</filename> and 
+           <filename>caasp-environment.yaml</filename> Heat templates to your 
+           workstation and then run the <command>openstack stack create</command>
+           command.
+          </para>
+          <screen>
+&prompt.user;<command>openstack stack create \ 
+-t caasp-stack.yaml \
+-e caasp-environment.yaml \
+--parameter image=<replaceable>CaaSP-3</replaceable> <replaceable>caasp3-stack</replaceable></command>
+          </screen>
+         </step>
+
+        <step>
+         <para>
+          Find out which (external) IP address was assigned to the admin node
+          of your &productname; cluster (here: <literal>10.81.1.51</literal>).
+         </para>
+          <screen>
+&prompt.user;<command>openstack server list --name "admin" | awk 'FNR > 3 {print $4 $5 $9}'</command>
+caasp3-stack-admin|10.81.1.51
+         </screen>
+        </step>
+        <step>
+         <para>
+          Visit the external IP in your browser. You should see the &dashboard; 
```

Comment:
"IP address" please :) "IP" and "IP address" are not the same.

Reply:
done